Engine/Component/ComponentParticleSystem.cpp

```diff
@@ -3,12 +3,11 @@
 #include "Main/Application.h"
 #include "Module/ModuleEffects.h"
 #include "Module/ModuleProgram.h"
+#include "Module/ModuleResourceManager.h"
```

Comment:
I only moved the include, so didn't know where it was used. Actually, if you delete the include, there's several points that break (related to brofiler, owner and billboards).

Reply:
Yeah, without the line it doesn't compile. But the thing is because we are not using the resource manager there is no need to included it, and then include the missing stuff. This way the code would be more consistent and we wouldn't be adding the extra includes defined inside `Module/ModuleResourceManager.h`